recipe/meta.yaml

```diff
@@ -10,23 +10,22 @@ source:
   sha256: d424dc35ebf2b3d3a36f4c76bc6bf8fdc635ebbbe5aa2953a091ef0baa8a8086
 
 build:
-  number: 0
-  features:
-    - vc14  # [win and py>=35]
-  skip: true  # [win and py<35]
+  number: 1
+  skip: true  # [win and vc<14]
```

Comment:
Maybe you need to not skip the vc<14? Or change that to target the Python version?

Reply:
vc9 is not supported as the lib requires c++14, so only py3.5 and py3.6 are built for windows

Reply:
Sure, but I think that means you can't support all versions of Python, because Python is tied to specific versions of vc.  So you might want to try skipping old versions of python explicitly.

Reply:
yes, it follows that py2.7 is skipped